erratatool.yml

```diff
@@ -22,7 +20,7 @@ solution: |
 
   For OpenShift Container Platform 3.11 see the following documentation, which will be updated shortly for this release, for important instructions on how to upgrade your cluster and fully apply this asynchronous errata update:
 
-  https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html
+    https://docs.openshift.com/container-platform/3.11/upgrading/index.html
```

Comment:
The release notes include the changes in each release along with a link to the upgrading docs, so I think this is correct as-is.  (Note that all other branches also point to release notes here.)  wdyt @sosiouxme?